pyglmnet/utils.py

```diff
@@ -4,40 +4,13 @@
 
 import numpy as np
 from copy import copy
-from scipy.stats import norm
 import logging
 
 
 logger = logging.getLogger('pyglmnet')
 logger.addHandler(logging.StreamHandler())
 
 
-def probit(z):
```

Comment:
@pavanramkumar you will notice this trend that I like deleting lines a lot ;)

Quoting zen of python: "Flat is better than nested." We don't really need these wrappers and in any case I don't see why they had to be public ...